src/topp/FLASHDeconv.cpp

```diff
@@ -654,16 +617,25 @@ class TOPPFLASHDeconv : public TOPPBase
 
         for (auto& pg : fd_charge_decoy.getDeconvolvedSpectrum())
         {
+          if(!write_detail){
```

Comment:
please adhere to the coding standards
```suggestion
          if (!write_detail)
          {
```